app/src/main/res/layout/view_calculator_small.xml

```diff
@@ -34,7 +34,9 @@
             android:paddingRight="@dimen/image_padding"
             android:paddingEnd="@dimen/image_padding"
             android:background="?android:selectableItemBackground"
-            android:src="@drawable/ic_copy_small" />
+            android:scaleX="0.75"
```

Comment:
android:width="18dp" does not work here

Reply:
What is the reason?

Reply:
I do not know. I just see that the icon stays the same size weather I specify android:width / android:height or not.

Reply:
Can you please investigate this a little bit more. Maybe you need to use `android:scaleType`?

Reply:
How about this:
```
android:width="18dp"
android:height="18dp" 
android:scaleType="fitCenter"
```

Reply:
Does not work. I tried this and all other options of scaleType without success. I think it is because width and height are set inside the drawable xml ...
But why don't you like the scaleX/scaleY? We know that the source is 24dp. So the scale is always correct.

Reply:
No, this is not the reason. The reason is, that `android:width`and `android:height` are no valid attributes for `<ImageView ... />`. So these are simply ignored.

Reply:
I would keep the `scaleX/Y` attribute. Creating two drawables with 24dp and one with 18dp is in my point of view no good option.

Reply:
What about `android:layout_width` and `android:layout_height`?

Reply:
We already have this. Setting `android:layout_width` works but `android:layout_width="18dp"` again creates a very small icon, so we have to use `android:layout_width="26dp"` (increased by the padding). Do you really want this?

Reply:
OK, let's leave scale for now. Is there anything else remaining?

Reply:
No, that's it.